src/components/HeroHome/_HeroHome.scss

```diff
@@ -1,3 +1,19 @@
+.coa-HeroHome-container {
+  img {
+    position: absolute;
+    z-index: -1;
+    height: 26rem;
+    width: 100%;
+    object-fit: cover;
```

Comment:
do we need a polyfill for this? poor support for this in IE & Edge.
https://caniuse.com/#search=object-fit

Reply:
Found these:
https://github.com/bfred-it/object-fit-images/
https://github.com/jonathantneal/fitie

Any preferences?

Reply:
Created issue #243 to address this